### PR TITLE
Fix progress styling

### PR DESCRIPTION
**Pull request by korneliakobiela**
Issue: #325
Problem: Bad styling of progress element
Solution: Change selector and classes of elements

Signed-off-by: Kornelia Kobiela <k.kobiela@samsung.com>

### CODE REVIEW

File: tau-component-packages/components/progress/package.json [hunk 1]
```diff
@@ -81,6 +81,6 @@
       "thickness": "null"
     }
   },
-  "resizable": true,
+  "resizable": false,
```
Comment (singa2000) [inline tau-component-packages/components/progress/package.json]: Why it need to set the resizable to false?
So we can't resize the progress widget?

Comment (korneliakobiela) [reply tau-component-packages/components/progress/package.json]: Now resizing crashes progress. I've found some additional issues, I will write it on GH, and fix 
image

Comment (korneliakobiela) [reply tau-component-packages/components/progress/package.json]: Issue #369 is about it. Until fix it, we decided to turn off resizing

Comment (singa2000) [reply tau-component-packages/components/progress/package.json]: ok. next time, fix it please.

**Pull request merged**